### PR TITLE
Update chart for Keda 2.4 API

### PR DESCRIPTION
**Pull request by jasonpaige**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: function/values.yaml [hunk 1]
```diff
@@ -14,16 +14,15 @@ dnsConfig:
...
-cooldownPeriod:  300 # Optional. Default: 300 seconds
-minReplicaCount: 0   # Optional. Default: 0
-maxReplicaCount: 5 # Optional. Default: 5
+  parallelism: 1              # [max number of desired pods](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#controlling-parallelism)
+  completions: 1              # [desired number of successfully finished pods](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#controlling-parallelism)
+  activeDeadlineSeconds: 600  # Specifies the duration in seconds relative to the startTime that the job may be active before the system tries to terminate it; value must be positive integer
+  backoffLimit: 6             # Specifies the number of retries before marking this job failed. Defaults to 6
+pollingInterval: 30           # Optional. Default: 30 seconds
+maxReplicaCount: 5            # Optional. Default: 5
+successfulJobsHistoryLimit: 1 # Optional. Default: 100. How many completed jobs should be kept.
```
Comment (adusumillipraveen) [inline function/values.yaml:24]: How about 3?
```suggestion
successfulJobsHistoryLimit: 3 # Optional. Default: 100. How many completed jobs should be kept.
```

File: function/templates/_servicebus-trigger.yaml [hunk 2]
```diff
@@ -9,6 +9,6 @@
     topicName: {{ .topicName }}
     subscriptionName: {{ .subscriptionName }}
     {{- end }}
-    connection: {{ .connection }}
-    queueLength: {{ .queueLength | quote }}
+    connectionFromEnv: {{ .connection }}
+    messageCount: {{ .queueLength | quote }}
```
Comment (adusumillipraveen) [inline function/templates/_servicebus-trigger.yaml:13]: Should we rename these and make a breaking change so its easy to refer Keda docs ?
```suggestion
    connectionFromEnv: {{ .connectionFromEnv }}
    messageCount: {{ .messageCount | quote }}
```

**Review by adusumillipraveen — COMMENTED** [review]
minor suggestions

File: function/templates/_servicebus-trigger.yaml (hunk 2 above)
Comment (jasonpaige) [reply function/templates/_servicebus-trigger.yaml:13]: how about `connectionFromEnv: {{ coalesce .connectionFromEnv .connection }}` to avoid the BC?

Comment (adusumillipraveen) [reply function/templates/_servicebus-trigger.yaml:13]: perfect, assuming it gives preference to connectionFromEnv if both present .

**Pull request closed without merging**